REFERENCE.md

```diff
@@ -1,459 +0,0 @@
-
```

Comment:
would be nice to keep this separate file, just update with docs from the 3.0 branch